<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>404</width>
    <height>311</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: black;</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>20</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
   <property name="text">
    <string>Авторизация</string>
   </property>
  </widget>
  <widget class="QPushButton" name="log_in">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>240</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color: rgb(36,36,36);
color: white;
border-radius: 10px;
}
QPushButton:hover {
  background-color: rgb(100,100,100);
color: white;
border-radius: 10px;
}
QPushButton:pressed {
   background-color: rgb(50,50,50);
color: white;
border-radius: 10px;
}
QPushButton:disabled {
   background-color: rgb(70,70,70);
color: white;
border-radius: 10px;
}

</string>
   </property>
   <property name="text">
    <string>Вход</string>
   </property>
  </widget>
  <widget class="QPushButton" name="reg">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>240</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color: rgb(36,36,36);
color: white;
border-radius: 10px;
}
QPushButton:hover {
  background-color: rgb(100,100,100);
color: white;
border-radius: 10px;
}
QPushButton:pressed {
   background-color: rgb(50,50,50);
color: white;
border-radius: 10px;
}
QPushButton:disabled {
   background-color: rgb(70,70,70);
color: white;
border-radius: 10px;
}

</string>
   </property>
   <property name="text">
    <string>Регистрация</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="login">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>321</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(36,36,36);
color: white;
border-radius: 5px;</string>
   </property>
   <property name="text">
    <string>Логин</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>321</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(36,36,36);
color: white;
border-radius: 5px;</string>
   </property>
   <property name="text">
    <string>Пароль</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
